pkg/controllers/vdb/httpservercertgen_reconciler.go

```diff
@@ -59,15 +59,15 @@ func (h *HTTPServerCertGenReconciler) Reconcile(ctx context.Context, _ *ctrl.Req
 	}
 	// If the secret name is set, check that it exists. As a convenience we will
 	// regenerate the secret using the same name.
-	if h.Vdb.Spec.NmaTLSSecret != "" {
-		nm := names.GenNamespacedName(h.Vdb, h.Vdb.Spec.NmaTLSSecret)
+	if h.Vdb.Spec.NMATLSSecret != "" {
+		nm := names.GenNamespacedName(h.Vdb, h.Vdb.Spec.NMATLSSecret)
 		secret := corev1.Secret{}
 		err := h.VRec.Client.Get(ctx, nm, &secret)
 		if errors.IsNotFound(err) {
 			h.Log.Info("httpServerTLSSecret is set but doesn't exist. Will recreate the secret.", "name", nm)
```

Comment:
Can we update the message to indicate the name of the secret?
```suggestion
			h.Log.Info("nmaTLSSecret is set but doesn't exist. Will recreate the secret.", "name", nm)
```